package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name"        : "sharp-image-loader",
-  "version"     : "0.1.0-alpha.2",
+  "version"     : "0.1.0-alpha.3",
```

Comment:
You can leave this out - this will automatically be done when I release a new version.